tests/coverage_lcov/test_lcov.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+function outerdir(){
+   echo "$1" | sed -E 's@/[^.]+.*@/@'
+}
+DIR_BUILD=$PWD
+DIR_OUTER=$(outerdir "$0")
+if [ "$DIR_OUTER" = "./" ]
+then
+     echo "The script must be run from the 'facelift/build'"
```

Comment:
The script should be runnable from anywhere, especially from folders which are outside of the source folder since we do not want to write build files into the source folder.